lib/agent/actions/wipe/windows.js

```diff
@@ -2,8 +2,35 @@ var os   = require('os'),
     join = require('path').join,
     exec = require('child_process').exec;
 
+var outlook_version = join('HKEY_CLASSES_ROOT', 'Outlook.Application', 'CurVEr');
+var profileRegistry = join('HKEY_CURRENT_USER', 'Software', 'Microsoft');
+
+var getOutlookVersion = function(cb) {
+  exec('reg query ' + '"' + outlook_version + '"', function(err, stdout) {
```

Comment:
I'd move all the reg operations to their own functions, for instance:

var registry = {
  query: function() { ... },
  delete: function() { ... },
  add: function() { ... }
}